src/ensembl/src/content/app/entity-viewer/gene-view/components/transcripts-table/TranscriptsTable.scss

```diff
@@ -0,0 +1,93 @@
+@import 'src/styles/common';
+
+.table {
+  display: grid;
+  grid-template-columns: 1fr 3fr 1fr 1fr 2fr;
```

Comment:
here and elsewhere: we might be better off actually specifying in pixels the widths of individual columns